### ISSUE TITLE
Accessibility issues on 2.5.0 Insiders (validation bug)

### ISSUE THREAD

**Issue by Shobhit1** [microsoft/accessibility-insights-web#805]
## Describe the bug

Found an a11y issue while validating Insiders 2.5.0 version.
Rule failed: `label-content-name-mismatch`

**To Reproduce**

1. Go to Assessment View from the extension for any target page
2. Open developer tools (F12) and type `A11yselfvalidator.validate()`
3. See error

## Expected behavior

`A11ySelfvalidator` should return `[]` on Assessment details view page.

## Screenshots

![image](https://user-images.githubusercontent.com/4496335/59540390-800aea80-8eb3-11e9-8404-615755e6b437.png)

## Extension (please complete the following information)

- OS: Mac
- Version: Insider 2.5.0
- Browser Version: Microsoft Edge Version 76.0.161.0 (Official build) Canary (64-bit)
- Target Page: https://www.washington.edu/accesscomputing/AU/before.html

## Are you willing to submit a PR?

Yes.

## Did you search for similar existing issues?
Yes

**Comment by ferBonnin** [comment]
3/4 issues are related to this https://github.com/dequelabs/axe-core/issues/1635
the other one (link "see more info here") is a real issue.

**Comment by Shobhit1** [comment]
We partially fixed some of them and there is a bug on axe-core for a false positive for a rule in WCAG 2.1. So, I feel we can close this bug.
Reference bug: https://github.com/dequelabs/axe-core/issues/1635

**Comment by ferBonnin** [comment]
@Shobhit1 agree to close this issue.

**Comment by Shobhit1** [comment]
Closing this one as fixed with https://github.com/microsoft/accessibility-insights-web/tree/v2.5.4

**Comment by ferBonnin** [comment]
since this is closed marking it as resolved
